apps/admin-gui/src/app/shared/components/role-filter-inputs/group-roles-filter.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'app-group-roles-filter',
+  templateUrl: './group-roles-filter.component.html',
+  styleUrls: ['./group-roles-filter.component.scss'],
+})
+export class GroupRolesFilterComponent {
+  @Output()
+  filterRoles = new EventEmitter<string[]>();
+  @Output()
+  filterRoleTypes = new EventEmitter<string[]>();
+  roles = new FormControl(['']);
+  roleTypes = new FormControl(['']);
+  rolesList = ['GROUPADMIN', 'GROUPOBSERVER', 'GROUPMEMBERSHIPMANAGER'];
+  roleTypesList = ['DIRECT', 'INDIRECT'];
```

Comment:
As mentioned would prefer if this were not strings.

Reply:
https://github.com/CESNET/perun/pull/4105#discussion_r1345294470